V6_MIGRATION.md

```diff
@@ -149,23 +148,22 @@ The `BraintreeUnionPay` module, and all containing classes, was removed in v6. U
 Now, you can tokenize just with the card details:
 
 ```
-let braintreeClient = BTAPIClient(authorization: "<CLIENT_AUTHORIZATION>")!
-let cardClient = BTCardClient(apiClient: braintreeClient)
+let cardClient = BTCardClient(apiClient: <MY_BTAPIClient>)
 
 let card = BTCard()
 card.number = "4111111111111111"
 card.expirationMonth = "12"
 card.expirationYear = "2025"
 
-cardClient.tokenizeCard(card) { (tokenizedCard, error) in
+cardClient.tokenize(card) { tokenizedCard, error in
```

Comment:
Below this we have `// Communicate the tokenizedCard.nonce to your server, or handle error` but in the rest of this file we just do `// handle response`. Do folks have a preference on this? We should update them all to be the same, imo.

Reply:
Updated to `handle response`: https://github.com/braintree/braintree_ios/pull/984/commits/5842ae352f8324be34cc5c372319765e22a2aa72